### PR TITLE
Add Changelog for v1.0.0 RC1

### PR DESCRIPTION
**Pull request by 6543**
add changelog for the v1.0.0 release candidate

### CODE REVIEW

**Comment by lafriks** [comment]
Can we remove groups and change to depenends on in pipeline format before releasing 1.0?

**Comment by 6543** [comment]
@lafriks well a reason to have a v2 when the pull dod arive?

**Comment by 6543** [comment]
Or is it urgent and do block us from spething specific?

**Comment by 6543** [comment]
@lafriks I created https://github.com/woodpecker-ci/woodpecker/issues/1860 ... as we did not hat an dedicated issue for it just mentions in the pipeline-rfc

**Comment by 6543** [comment]
As for me ... we have enough fixes and features to be released ... but I'll wait until you give the release a go (just if you have a pull for it already)

**Comment by lafriks** [comment]
I don't have PR, I'm fine with also releasing it in 2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,181 @@
...
+  * Fix branch condition on tags (#917)
+  * Use asymmetric key to sign webhooks (#916)
+  * Add agent tagging / filtering for pipelines (#902)
+  * Delete old fallback for "drone.sqlite" (#791)
+  * Migrate to certmagic (#360)
+* FEATURES
+  * Implement YAML Map Merge, Overrides, and Sequence Merge Support (#1720)
+  * Add users UI for admins (#1634)
+  * Add agent no-schedule flag (#1567)
+  * Secrets encryption in database (#1475)
```
Comment (6543) [inline CHANGELOG.md:27]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,181 @@
...
+  * Hide log output container if step does not have logs (#1086)
+  * Fix to show build pipeline parse error (#1066)
+  * Pipeline compiler should not alter specified image (#1005)
+  * Gracefully handle non-zero exit code in local backend (#1002)
+  * Replace run_on references with runs_on (#965)
+  * Set default logging value of CLI to info (#871)
+  * Support conditional branch as an array to align with documentation (#836)
+  * Fix redirect after login (#824)
+* ENHANCEMENTS
+  * Drop old columns (#1838)
```
Comment (6543) [inline CHANGELOG.md:90]:
```suggestion
  * Added Woodpecker Host Config used for Webhooks (#1869)
  * Drop old columns (#1838)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,196 @@
 # Changelog
 
+## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-06-X
```
Comment (6543) [inline CHANGELOG.md:3]:
```suggestion
## [1.0.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/1.0.0-rc1) - 2023-07-XX
```

**Review by lafriks — APPROVED** [review]

**Pull request merged**